pkg/seccomp/seccomp.json

```diff
@@ -399,6 +400,86 @@
 			"includes": {},
 			"excludes": {}
 		},
+		{
+			"names": [
+				"add_key",
+				"bdflush",
+				"break",
+				"cachectl",
+				"clone3",
+				"ftime",
+				"futex_time64",
+				"get_kernel_syms",
+				"getpmsg",
+				"gtty",
+				"idle",
+				"io_pgetevents",
+				"io_pgetevents_time64",
+				"io_uring_enter",
+				"io_uring_register",
+				"io_uring_setup",
+				"kexec_file_load",
+				"kexec_load",
+				"membarrier",
+				"migrate_pages",
+				"move_pages",
+				"mpx",
+				"mq_timedreceive_time64",
+				"mq_timedsend_time64",
+				"multiplexer",
+				"nfsservctl",
+				"nice",
+				"oldfstat",
+				"oldlstat",
+				"oldolduname",
+				"oldstat",
+				"olduname",
+				"pciconfig_iobase",
+				"pciconfig_read",
+				"pciconfig_write",
+				"pkey_alloc",
+				"pkey_free",
+				"pkey_mprotect",
+				"prof",
+				"profil",
+				"putpmsg",
+				"readdir",
+				"recvmmsg_time64",
+				"request_key",
+				"rseq",
+				"rtas",
+				"rt_sigtimedwait_time64",
+				"sched_rr_get_interval_time64",
+				"security",
+				"semtimedop_time64",
+				"sgetmask",
+				"spu_create",
+				"spu_run",
+				"ssetmask",
+				"stty",
+				"subpage_prot",
+				"swapcontext",
+				"swapoff",
+				"swapon",
+				"switch_endian",
+				"_sysctl",
+				"sys_debug_setcontext",
+				"sysfs",
+				"sysmips",
+				"timer_settime64",
+				"tuxcall",
+				"ulimit",
+				"uselib",
+				"userfaultfd",
+				"ustat",
+				"vm86",
+				"vm86old",
+				"vmsplice",
+				"vserver"
+			],
+			"action": "SCMP_ACT_ERRNO",
+			"errnoRet": 1
```

Comment:
Should this be a magic number?

Reply:
I think it is already too late for errnoRet as we already support it.  Perhaps add a `errnoRetString` that can be used in place of `errnoRet`? 

We could make `defaultErrnoRet` a string, but I don't see a way to convert from a name to an errno list in the std library, so we would need to do the conversion ourselves.